h/static/scripts/annotator/guest.coffee

```diff
@@ -384,11 +411,10 @@ module.exports = class Guest extends Annotator
     event.preventDefault?()
     event.stopPropagation?()
     @adder.hide()
-    switch event.target.dataset.action
```

Comment:
Another weird mobile safari thing, the difference is `event.target.dataset.action` doesn't have to be set to comment. It's also one less line! : )